File: 242-ValidAnagram/242-ValidAnagram.java
// Last updated: 12/28/2025, 4:53:39 PM
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length()!=t.length()) return false;
        char [] sarray=s.toCharArray();
        char [] tarray=t.toCharArray();
        Arrays.sort(sarray);
        Arrays.sort(tarray);
        int c=0;
        for(int i=0;i<sarray.length;i++){
            if(sarray[i]!=tarray[i]) return false;
            else c++;
        }
        return true;
    }
}

File: 242-ValidAnagram/242-ValidAnagram.py
# Last updated: 12/28/2025, 4:53:40 PM
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if(len(s)!=len(t)):
            return False
        l=list()
        c=0
        for i in s:
            l.append(i)
        for i in t:
            if i in l:
                l.remove(i)
        if(len(l)==0):
            return True
        return False